src/TypesFinder/ResolveTypes.php

```diff
@@ -30,6 +30,7 @@ public function __invoke(array $stringTypes, Context $context) : array
             $resolvedTypes[] = $this->typeResolver->resolve($stringType, $context);
         }
 
+        /** @var Type[] */
```

Comment:
This doccomment should be on line 27 and contain variable name.

Reply:
I did one of those for-the-typechecker things where I told it that the array only gets non-nullable objects in the `foreach`